### PR TITLE
Restructure build process

### PR DESCRIPTION
**Pull request by tboerger**
Get the build process for the whole documentation more or less finished.

### CODE REVIEW

**Comment by settermjd** [comment]
I like the `[WIP]` convention in the title. It's far more obvious than the purple label.

File: docs/build-the-docs.md [hunk 1]
```diff
@@ -2,13 +2,12 @@
 
 ## Install Antora's Dependencies
 
-Before you can build the documentation, you need to install the required dependencies.
-To install them, you need to run `yarn install`.
-This will install all the dependencies specified in `package.json`.
-**Note:** If your environment already has GNU make installed, you can run `make install` instead.
+Before you can build the documentation, you need to install the required dependencies. To install them, you need to run `yarn install`. This will install all the dependencies specified in `package.json`.
```
Comment (settermjd) [inline docs/build-the-docs.md]: Why were these lines compacted together?

File: docs/build-the-docs.md [hunk 2]
```diff
@@ -30,30 +29,35 @@ To build the documentation using the Docker container, from the command line, in
...
 
 - Starts up [ownCloud's custom Antora Docker container](https://hub.docker.com/r/owncloudci/antora/)
 - Runs Antora's `generate` command, which regenerates the documentation
-- You can add the `--clean` option to clean the build directory of any leftover artifacts from the previous build, including PDF's.
+- You can add the `--pull` option to update the dependent repositories, or any other available flag.
 
-If all goes well, you will _not_ see any console output.
-If a copy of the container doesn't exist locally, you can pull down a copy, by running `docker pull owncloudci/antora:latest`.
-Otherwise, you should see output similar to the following:
+If all goes well, you will _not_ see any console output. If a copy of the container doesn't exist locally, you can pull down a copy, by running `docker pull owncloudci/antora:latest`. Otherwise, you should see output similar to the following:
```
Comment (settermjd) [inline docs/build-the-docs.md]: Why were these lines compacted together?

File: docs/build-the-docs.md [hunk 3]
```diff
@@ -65,106 +69,41 @@ latest: Pulling from owncloudci/antora
 d5a49762c0f9: Download complete
 ```
 
-#### Using Make
+#### Using Antora from the Command-Line
 
-Using Make, as in the example below, is the easiest way to build the documentation.
-The Makefile has a predefined target (`html`) which calls Antora, supplying all of the required options to build the docs, to build the documentation on the master branch of [the ownCloud documentation repository](https://github.com/owncloud/docs).
+Using Yarn, as in the example below, is the easiest way to build the documentation. This project has a predefined target (`antora`) which calls Antora, supplying all of the required options to build the docs, to build the documentation on any branch of [the ownCloud documentation repository](https://github.com/owncloud/docs).
```
Comment (settermjd) [inline docs/build-the-docs.md]: Why were these lines compacted together?

**Comment by tboerger** [comment]
> On the whole, the PR looks good. However, the PR appears to be a collection of a range of changes. It's easier to review if it relates to just one change.

Except the books content compact the changes are coupled. I have updated the docs matching the workflow.

> The last thing is that the lines between the includes in the PDF adoc files will break the generation of the PDFs. I don't remember off-hand why, but includes need to be separated by a new line for asciidoctor-pdf to work properly.

I can't confirm that, that PDF documents are building totally fine. The only thing where are some complains is about the admin and developer manual which prints a few warnings.

**Comment by settermjd** [comment]
But those warnings are likely going to break the generation of the PDF manuals. Please remove the changes to the book adoc files.

**Comment by tboerger** [comment]
The PDFs are built and as far as I can see it doesn't look incomplete?

**Pull request merged**